components/help-dropdown/help-dropdown.scss

```diff
@@ -0,0 +1,18 @@
+#help-dropdown {
```

Comment:
Let's not target using IDs; it makes overriding much harder later on if necessary.